### DISCUSSION TITLE
Automatically scan and delete comments

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#832] [Q&A]
Is it possible to set it to automatically scan and delete SPAM comemnts?

**Comment by Firecul** [comment]
Yes, using config files you can do that (deleting is on your own channel or one you have access to obviously)
There is no write-up atm for that feature but using the [config file](https://github.com/ThioJoe/YT-Spammer-Purge/wiki/Config-File-Settings) you can set the tool to run using settings you define in it without interaction while it's running. Then you just have to run the program on a schedule you wish using something like Task Scheduler or cronjobs.
